src/config/AOCODARCF7DUAL/config.h

```diff
@@ -95,7 +95,7 @@
 
 #define ADC1_DMA_OPT        0
 
-//TODO #define GYRO_TO_USE SECOND
+#define GYRO_CONFIG_USE_GYRO_DEFAULT GYRO_CONFIG_USE_GYRO_2
```

Comment:
Can you rename this to DEFAULT_GYRO_TO_USE

Reply:
Done.